apps/fluent-tester/src/TestComponents/Tooltip/consts.ts

```diff
@@ -0,0 +1,2 @@
+export const HOMEPAGE_TOOLTIP_BUTTON = 'Homepage_Tooltip_Button';
```

Comment:
Do we need this and the consts file in the E2E folder? They contain the same code